WalletWasabi.Gui/Converters/NetworkColorConverter.cs

```diff
@@ -0,0 +1,25 @@
+using Avalonia;
+using Avalonia.Data.Converters;
+using Avalonia.Media;
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using WalletWasabi.Exceptions;
+
+namespace WalletWasabi.Gui.Converters
+{
+	public class NetworkColorConverter : IValueConverter
+	{
+		public object Convert(object value, Type targetType, object parameter, CultureInfo culture) =>
+			value switch
+			{
+				Network network => Application.Current.Resources[network.Name],
+				_ => Application.Current.Resources[Network.Main.Name]
```

Comment:
Is this wise? There are only three networks at the moment. If there is a new one, this should crash so that it is known that this code is not ready for new network.

I don't expect any new networks but this seems to me is hiding problems.

Reply:
This syntax is new to me. Will it crash? _ seems to be the default case, isn't it?

Reply:
Yes, `_` is an equivalent of `default` in switch.

It won't crash and I think it should.

Reply:
Yes, it will crash.

Reply:
Ah, I see what you mean. 

* I think it should crash whenever the network in unknown in `Application.Current.Resources`
* It should crash when `value` is not `Network`

Reply:
As this is only for color selection, IMO this is not the part it should crash when there is a problem with the network. So I made it failsafe and add a constant fallback value.

Reply:
According to our discussion, I reverted back to Lucas' solution. The app should crash on an invalid network parameter.